docs/stability.md

```diff
@@ -18,21 +18,25 @@ Things considered stable for 2.x:
 * Configuration file format (minus the service discovery remote read/write, see below)
 * Rule/alert file format
 * Console template syntax and semantics
-* Remote write sending, per the [1.0 specification](https://prometheus.io/docs/concepts/remote_write_spec/).
+* Remote write sending, per the [1.0 specification](https://prometheus.io/docs/concepts/remote_write_spec/) and receiving
+* Agent mode
+* OTLP receiver endpoint
 
-Things considered unstable for 2.x:
+Things considered unstable for 3.x:
 
 * Any feature listed as experimental or subject to change, including:
-  * The [`holt_winters` PromQL function](https://github.com/prometheus/prometheus/issues/2458)
-  * Remote write receiving, remote read and the remote read endpoint
+  * The [`double_exponential_smoothing` PromQL function](https://github.com/prometheus/prometheus/issues/2458)
+  * Remote remote read and the remote read endpoint
```

Comment:
Should be `Remote read and the remote read endpoint`

Extra `remote` in there.